arithmetization/src/main/java/net/consensys/linea/zktracer/runtime/callstack/CallFrame.java

```diff
@@ -71,7 +71,6 @@ public boolean isMessageCall() {
   @Getter private int byteCodeDeploymentNumber;
   private EWord eCodeAddress = null; // memoization
   @Getter private Bytecode code = Bytecode.EMPTY;
-  @Getter private int codeFragmentIndex = -1;
```

Comment:
Isn't this desirable ? A sort of sanity check ?

Reply:
it wasn't used at all, as we're using a function called `getCodeFragmentIndex()`